news/2018-04-23-twc-quarters-recap.md

```diff
@@ -0,0 +1,114 @@
+
+
+### Group Stage Matches
+
+The Group Stage started off with Australia setting the tone for the weekend. In spite of Finland putting up a valiant fight and winning a point, **they were beaten convincingly just like the other low seeded teams would be.** 
+
+After a few more matches, United Kingdom came out swinging against France. In fact, they swang hard enough to find themselves on match point with the opportunity to clinch the game 4-2 over the French. Unfortunately for the brits, **France pulled themselves together to come back and win on tiebreaker.** The fierce battle between the powerhouses goheegy and TimmyAkmed was a delight to watch, but the upsets would have to wait until later.
+
+The nocturnal matches (UTC time) both resulted in upsets. **Chile and South Korea showed a dominant performance against the top seeds Australia and Brazil respectively**, only losing 1 map each. Going into the Round Of 16, this put Chile and South Korea in even, winnable matchups while Australia and Brazil had tough teams ahead: Taiwan and USA.
+
+
+The third and final upset of the weekend, Spain versus Indonesia, was a real thriller. After trading close points with each other for 6 maps, the two teams ended up on tiebreaker where **Spain clutched out the victory.** 
+![](/wiki/shared/news/2018-04-19-twc-ro16-recap/spain-indo-tiebreaker.jpg)
+
+
+For the Round Of 16, this meant that Indonesia would face their fellow top seed France while Spain faced the United Kingdom. Both of those were rematches from last year where France and Spain came out on top, so revenge-driven performances from Indonesia and United Kingdom were to be expected.
+
+
+### Group Stage Mappool
+
+**The nomod picks were the clear favourites for the teams in Groups this year.** One of the most picked maps was Firce777's classic memorization map [Yuitsuki yukari - Shiawase ni nareru kakushi komando ga aru rashii](https://osu.ppy.sh/beatmapsets/219733/#taiko/514465), which is a type of map that has not been too popular historically. Instead, the outlier in nomod was the accuracy map [Chroma - Collapse](https://osu.ppy.sh/beatmapsets/691352#taiko/1463029) which surprisingly only got picked twice. 
+
+The rejection of accuracy maps in this stage was made even more apparent by the bans. The OD7 Hidden map [Ice - Entrance](https://osu.ppy.sh/beatmapsets/755569#taiko/1590333) and the long DoubleTime pick [FELT - Day After](https://osu.ppy.sh/beatmapsets/323180#taiko/718430) combined made up almost half of all Group Stage bans. 
+
+A word on the picks/bans by [Nwolf](https://osu.ppy.sh/users/1910766), one of the map selectors: 
+
+*In Group Stage picks and bans mostly went as expected. We were surprised about the huge amount of picks on Shiawase and Goin' Under which we thought would be banned (and Caramel Custard which we thought would get ignored a bit more because it's technical and risky), and the high amount of bans on Entrance, which we considered the easier HD map.* 
```

Comment:
surprised by.
There's also a lot of "which we" repeats, but given it's quoting Nwolf directly, I don't know if it should be changed